sources/packages/backend/apps/api/src/route-controllers/assessment/_tests_/e2e/assessment.aest.controller.manualReassessment.e2e-spec.ts

```diff
@@ -0,0 +1,236 @@
+  });
+
+  it("Should create a manual reassessment when requested.", async () => {
+    // Arrange
+    const application = await saveFakeApplication(db.dataSource);
+    application.currentAssessment.studentAssessmentStatus =
+      StudentAssessmentStatus.Completed;
+    application.studentAssessments = [application.currentAssessment];
+    // Add an appeal to the application and current assessment/original assessement
+    const approvedAppealRequest = createFakeStudentAppealRequest();
+    const approvedAppeal = createFakeStudentAppeal({
+      application,
+      studentAssessment: application.currentAssessment,
+      appealRequests: [approvedAppealRequest],
+    });
+    const studentAppeal = await db.studentAppeal.save(approvedAppeal);
+    application.currentAssessment.studentAppeal = studentAppeal;
+    await db.application.save(application);
+
+    const endpoint = `/aest/assessment/application/${application.id}/manual-reassessment`;
+    const payload = { note: "Testing manual reassessment note." };
+    const token = await getAESTToken(AESTGroups.BusinessAdministrators);
+
+    // Act/Assert
+    let responseReassessmentId;
+    await request(app.getHttpServer())
+      .post(endpoint)
+      .send(payload)
+      .auth(token, BEARER_AUTH_TYPE)
+      .expect(HttpStatus.CREATED)
+      .then((response) => {
+        responseReassessmentId = response.body.id;
+        expect(responseReassessmentId).toStrictEqual(expect.any(Number));
+      });
+
+    const applicationFromDb = await db.application.findOne({
+      select: {
+        id: true,
+        currentAssessment: {
+          id: true,
+          offering: { id: true },
+          studentAppeal: { id: true },
+          triggerType: true,
+          studentAssessmentStatus: true,
+        },
+        studentAssessments: {
+          studentAssessmentStatus: true,
+          offering: { id: true },
+          studentAppeal: { id: true },
+        },
+        isArchived: true,
+        applicationStatus: true,
+      },
+      relations: {
+        currentAssessment: { offering: true, studentAppeal: true },
+        studentAssessments: {
+          offering: true,
+          studentAppeal: true,
+        },
+      },
+      where: {
+        id: application.id,
+        studentAssessments: {
+          triggerType: AssessmentTriggerType.OriginalAssessment,
+        },
+      },
+    });
+    const [originalAssessment] = applicationFromDb.studentAssessments;
+    const manualAssessment = applicationFromDb.currentAssessment;
+    expect(manualAssessment.id).toBe(responseReassessmentId);
+    expect(manualAssessment.triggerType).toBe(
+      AssessmentTriggerType.ManualReassessment,
+    );
+    expect(manualAssessment.studentAssessmentStatus).toBe(
+      StudentAssessmentStatus.Submitted,
+    );
+    expect(manualAssessment.offering.id).toBe(originalAssessment.offering.id);
+    expect(manualAssessment.studentAppeal.id).toBe(
+      originalAssessment.studentAppeal.id,
+    );
```

Comment:
Instead of doing expect for individual properties, we can do an expect on manualAssessment object